the-collab-lab/website
# feat: select generic avatar from possible color veariations

## Summary

As it says on the tin!

## Test plan

1. Open [the Volunteers page on the deploy preview](https://deploy-preview-13--tcl-site-next.netlify.app/volunteer)
2. Verify that different generic avatars appear in the grid (as in the attached screenshot)
3. Open [the developers page](https://deploy-preview-13--tcl-site-next.netlify.app/developers)
4. Verify that developers without avatars have the new pink avatar 1️⃣ 

1️⃣ This happens because some of our Collabies in the CMS, developers especially, have _a hard-coded reference to `generic-avatar.png`_ in the CMS, rather than _no_ avatar string. I made the pink avatar's filename `generic-avatar.png` to support this case.

## Screenshot
<img width="647" alt="CleanShot 2022-12-20 at 13 45 56@2x" src="https://user-images.githubusercontent.com/13525251/208773255-64723758-40a0-4d85-b1a0-8bb407e22833.png">
